lxd/instance/drivers/qmp/commands.go

```diff
@@ -247,6 +247,88 @@ func (m *Monitor) SetMemoryBalloonSizeBytes(sizeBytes int64) error {
 	return m.run("balloon", args, nil)
 }
 
+// AddBlockDevice adds a block device.
+func (m *Monitor) AddBlockDevice(blockDev map[string]interface{}, device map[string]string) error {
+	revert := revert.New()
+	defer revert.Fail()
+
+	if blockDev != nil {
+		err := m.run("blockdev-add", blockDev, nil)
+		if err != nil {
+			return fmt.Errorf("Failed adding block device: %w", err)
+		}
+
+		revert.Add(func() {
+			blockDevDel := map[string]interface{}{
+				"node-name": blockDev["devName"],
+			}
+
+			err = m.run("blockev-del", blockDevDel, nil)
```

Comment:
You can just call `m.run("blockev-del", blockDevDel, nil)` here, no need to test for the result.

Reply:
Right. Also, there's a typo which needs to be fixed :)